MANIFEST.in

```diff
@@ -23,6 +23,7 @@ include COPYING
 include *.rst
 include *.sh
 include pytest.ini
+include docs/openapi.json
```

Comment:
We need to include this file in the repository, otherwise RTFD wouldn't build the documentation.